hs_core/tasks.py

```diff
@@ -795,13 +795,13 @@ def update_web_services(services_url, api_token, timeout, publish_urls, res_id):
 
             except Exception as e:
                 logger.error(e)
-                return e
+                return json.dumps(e.__dict__)
```

Comment:
I think `raise` would be more appropriate. The exception can still be caught and logged.  Celery shouldn't have an issue handling an exception that occurs within the task.

Reply:
Good call thanks @sblack-usu. I revised.